### PR TITLE
Pair Programming Contact Form

### PR DESCRIPTION
**Pull request by Kccendana**
Here are the things we did in this project:

- Created a contact form with the correct validation
- Collect data using Formspree
- Layout element using Flexbox

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -183,6 +183,47 @@ <h4 class="skill-name">Skills</h4>
                 </ul>
             </div>
         </section>
+        <section class="get-in-touch">
+            <div class="invitation">
+                <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            </div>
+            <form class="form"  action="https://formspree.io/f/xjvdqolb"
+            method="POST">
```
Comment (joskalenda) [inline index.html]: - [ ] Hey @team, check this out, your form is not active as says this error, please fix the issue, make your form spree submitting fine, and request another review

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @Kccendana , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
| IMAGE |

## Highlights

- [ ]... ✅ Good design
- [ ].. ✅ Green linters

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -183,6 +183,47 @@ <h4 class="skill-name">Skills</h4>
...
+                    class="form-input"
+                    placeholder="Email address"
+                    required
+                  />
+                  <textarea
+                  class="input-text"
+                  name="message"
+                  maxlength="500"
+                  placeholder="Write me something..."
+                ></textarea>
```
Comment (LionRouge1) [inline index.html:212]: [OPTIONAL]
- [ ] In the project requirement it states that all the form shouldn't submit when one or more field are empty. Therefore I suggest you add `required` attribute to the `textarea` to make the message field required as requested in the project requirement.

_**This is a requirement of the project but first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change**_

**Review by LionRouge1 — APPROVED** [review]
Hi @Kccendana & @Tupai566, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**